.babelrc

```diff
@@ -1,3 +1,4 @@
 {
-  "presets": [["@verdaccio"]]
+  "presets": [["@verdaccio"]],
+  "plugins": ["@babel/plugin-proposal-optional-chaining"]
```

Comment:
`@babel/plugin-proposal-optional-chaining` should be already accessible via our babel presets. No need to add a new dependency.

```
➜ yarn why @babel/plugin-proposal-optional-chaining
yarn why v1.21.1
warning ../../package.json: No license field
[1/4] 🤔  Why do we have the module "@babel/plugin-proposal-optional-chaining"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "@babel/plugin-proposal-optional-chaining@7.7.4"
info Reasons this module exists
   - "@verdaccio#babel-preset" depends on it
   - Hoisted from "@verdaccio#babel-preset#@babel#plugin-proposal-optional-chaining"
info Disk size without dependencies: "20KB"
info Disk size with unique dependencies: "68KB"
info Disk size with transitive dependencies: "68KB"
info Number of shared dependencies: 2
✨  Done in 0.78s.
```

Reply:
I added that because the test where not accepting this syntax. On closer review you are correct that this is part of the presets, but it seems the ```@babel/proposal-optional-chaining``` plugin is not loaded for the ```test``` env:
https://github.com/verdaccio/monorepo/blob/c81ef8b3efa22a0ae5f7ef7717325f8dc8fc09e5/tools/babel-preset/lib/builder.js#L55

I would suggest updating that line with:
```plugins: defaultPlugins.concat(transpiler.plugins, ['babel-plugin-dynamic-import-node', 'emotion']),```

Reply:
cc: @sergiohgz  <-- he is the master of the babel keys

Reply:
created PR here https://github.com/verdaccio/monorepo/pull/330